azure-pipelines.yml

```diff
@@ -19,6 +19,21 @@ pr: none
 jobs:
 - template: ci/build.yml
 
+- job: test_artifactory_credentials
```

Comment:
There isn't really a way to test the main branch credentials without merging this, but at least this will let us test it without having to make a release.